components/ntp_background_images/common/pref_util.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_NTP_BACKGROUND_IMAGES_COMMON_PREF_UTIL_H_
+#define BRAVE_COMPONENTS_NTP_BACKGROUND_IMAGES_COMMON_PREF_UTIL_H_
+
+class PrefService;
+
+namespace ntp_background_images::prefs {
+
+bool ShouldShowNewTabTakeoverInfobar(PrefService* prefs);
```

Comment:
Add `SuppressNewTabTakeoverInfobar` to set the value to 0 based on our conversation, after removing the feature.

Reply:
Fixed